modules/ROOT/pages/manage-monitor-applications.adoc

```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
 
 == Anypoint Monitoring
 
```

Comment:
ORG:
Ensure that the subject of this subtopic is introduced in the text of the topic that contains it, and that it is a peer of all other subtopics at this level.
Ensure also that the text of this subtopic introduces the subtopics that it contains.
GLOBAL org. recommendation for all deliverables.